manifests/init.pp

```diff
@@ -3,15 +3,15 @@
 # Manage inittab
 #
 class inittab (
-  $default_runlevel                   = 'USE_DEFAULTS',
-  $ensure_ttys1                       = undef,
-  $file_mode                          = '0644',
-  $require_single_user_mode_password  = false,
-  $enable_ctrlaltdel                  = true,
-  $ctrlaltdel_override_path           = 'USE_DEFAULTS',
-  $ctrlaltdel_override_owner          = 'root',
-  $ctrlaltdel_override_group          = 'root',
-  $ctrlaltdel_override_mode           = '0644',
+  Variant[Integer, String] $default_runlevel                 = 'USE_DEFAULTS',
```

Comment:
`USE_DEFAULTS` was a hack before hiera's data in module to pick the default value for a given platform. This should go away and just be replaced with hiera data. I've found that when this is very dynamic based on the platform, I make all the defaults in the class statement for the predominant platform, EL7.

Reply:
no worries, i'll see about updating the hiera data to have the standard defaults tonight